runtime/app/default-launcher.js

```diff
@@ -48,7 +48,7 @@ function launchApp (appDir, bridge, mode, options) {
   bridge.implement((force) => {
     if (force) {
       bridge.logger.info(`force stop process(${cp.pid}).`)
-      cp.kill(/** SIGKILL */9)
+      setTimeout(() => cp.kill(/** SIGKILL */9), 1000)
```

Comment:
Why do we delay 1000ms here?

Reply:
Leaves apps a gap period to freeing up resources.